app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js

```diff
@@ -188,8 +188,8 @@ define([
                 var addToCartButtonTextDefault = self.options.addToCartButtonTextDefault || $t('Add to Cart');
 
                 addToCartButton.removeClass(self.options.addToCartButtonDisabledClass);
-                addToCartButton.find('span').text(addToCartButtonTextDefault);
-                addToCartButton.attr('title', addToCartButtonTextDefault);
```

Comment:
It' not a good idea to remove the extension point for button title, the title may be redefined by 3d party extension via `self.options.addToCartButtonTextDefault` and will be broken after such changes